spec/jobs/emit_web_hook_event_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+require_dependency 'jobs/base'
+
+describe Jobs::EmitWebHookEvent do
+  before(:all) do
+    @post_hook = Fabricate(:web_hook)
```

Comment:
Read up on https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let